pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -164,8 +164,7 @@ def optimizer_step(
 
     def optimizer_zero_grad(self, current_epoch: int, batch_idx: int, optimizer: Optimizer, opt_idx: int) -> None:
         """Zeros all model parameter's gradients."""
-        model_ref = self.lightning_module
-        model_ref.optimizer_zero_grad(current_epoch, batch_idx, optimizer, opt_idx)
+        self.lightning_module.optimizer_zero_grad(current_epoch, batch_idx, optimizer, opt_idx)
```

Comment:
why not `_call_LM_hook`?

Reply:
Because it's called already with `_call_ttp_hook`.

https://github.com/PyTorchLightning/pytorch-lightning/blob/eec2bae6c984895d1d4df4e75cf628c58128928a/pytorch_lightning/loops/optimization/optimizer_loop.py#L403

No TTP subclasses this hook, so technically the optimizer_loop could call `_call_LM_hook` directly and this could be removed